src/Client.php

```diff
@@ -91,7 +91,7 @@ public function request($method, $path, $body = [])
 		}
 
 		try {
-			$result = $this->httpClient->request($url, $method, function (Event $event) use ($body) {
+			$result = $this->httpClient->request($method, $url, function (Event $event) use ($body) {
```

Comment:
he is right !
https://github.com/Understeam/yii2-httpclient/blob/master/src/Client.php#L182